project/Build.scala

```diff
@@ -1077,7 +1077,21 @@ object Build {
       }
     )
 
+  /** Packages the TASTy files of `stdlib-bootstrapped` in a jar */
+  lazy val `stdlib-bootstrapped-tasty` = project.in(file("stdlib-bootstrapped-tasty")).
+    withCommonSettings(Bootstrapped).
+    settings(
+      exportJars := true,
+      Compile / packageBin / mappings := {
+        (`stdlib-bootstrapped` / Compile / packageBin / mappings).value
+          .filter(_._2.endsWith(".tasty"))
+      },
```

Comment:
Those settings couldn't go directly in `stdlib-boostrapped`? Why is this additional project necessary?

Reply:
This is the solution you proposed to be able to test MiMa (classfiles) on `stdlib-bootstrapped` and also have a jar with TASTy only. 

An alternative would be to stop testing MiMa and only rely on TASTy MiMa.